resources/views/modals/confirmDeleteDeniedModal.php

```diff
@@ -1,29 +1,31 @@
 <!-- Main popup container with the form -->
-<div class="message-popup" id="confirmDeleteDeniedModal" style="display: none">
-    <!-- Close button in the top-right corner -->
-    <!-- <img src="https://img.icons8.com/?size=100&id=71200&format=png&color=1A1A1A" alt="Close" class="close-icon" id="closePopup"> -->
-    <a href="#" class="plain-action" id="closePopup" onclick="closeModal('confirmDeleteDeniedModal')"><i class="las la-times"></i></a>
+<div id="confirmDeleteDeniedModal" class="modal">
+    <div class="modal-content">
+    
+        <!-- Botón de cerrar -->
+        <span class="close-button" onclick="closeModal('confirmDeleteDeniedModal')">
+          <i class="fas fa-times"></i>
+        </span>
+       <!-- Título de la ventana -->
+        <h2>
+            <i class="fas fa-trash"></i>
+            ¿Desea eliminar las solicitudes denegadas?
+        </h2>
+        
+        <form action="/admin/delete/rejected/requests" method="POST">
+    
+            <div class="form-group">
+                <p>Esta acción eliminará los documentos de las solicitudes pero las cuentas relacionadas continuarán con el estado "activo".  Esta acción no se podrá revertir...</p>
+            </div>
 
-    <!-- Popup title -->
-    <h2 class="message-title">¿Desea eliminar las solicitudes denegadas?</h2>
-
-    <!-- Alert -->
-    <div class="message-options">
-        <h3> Esta acción eliminará los documentos de las solicitudes pero las cuentas relacionadas continuarán con el estado "activo".  Esta acción no se podrá revertir... </h3>
-    </div>
-
-    <!-- Form to delete rejected requests -->
-    <form action="/admin/delete/rejected/requests" method="POST">
-        <!-- CSRF Token -->
-
-
-    <!-- Buttons area -->
-    <div class="modal-actions">
-        <!-- Cancel button -->
-        <a href="#" class="secondary main-action-bright" onclick="closeModal('confirmDeleteDeniedModal')">Cancelar</a>
-
-        <!-- Confirm button -->
-        <button class="primary main-action-bright" onclick="closeModal('confirmDeleteDeniedModal')">Confirmar</button>
+            <div class="modal-actions">
+                <a class="main-action-bright" onclick="closeModal('confirmDeleteDeniedModal')">Cancelar</a>
+                <button type="submit" class="main-action-bright danger">
+                    <i class="las la-trash"></i> 
```

Comment:
change to font-awesome
```suggestion
                    <i class="fas fa-trash"></i> 
```